src/Components/OptionsList/OptionsList.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './OptionsList.css';
```

Comment:
You need to use either `import styles from './OptionsList.css';`or replace CSS loader. I would go with 
```js
import styles from './OptionsList.css';
```

and then you can use class names inside your components
```jsx
<ul className={styles.optionsList}>
...
</ul>
```

Reply:
Please check other components as well. It's a common issue in your PR